assets/user-profile/actions.ts

```diff
@@ -299,8 +299,8 @@ export function fetchFolders() {
         const userTopicsUrl = getFoldersUrl(state, false);
 
         return Promise.all([
-            state.company !== 'None' ? server.get(({_items}: Array<any>) => _items) : Promise.resolve([]),
-            server.get(({_items}: Array<any>) => _items),
```

Comment:
```
server.get(({_items}: Array<any>) => _items),
                   // ^ this type is for entire argument, not only what you destructure.
```

it should be `server.get(({_items}: {_items: Array<any>}) => _items),`

Reply:
Actually the URL to fetch from is missign too :D